_episodes/01-design.md

```diff
@@ -17,41 +17,49 @@ see [the instructor training course][training].
 
 ## Reverse Instructional Design
 
+### Idealized
+
 In principle,
 we design lessons in four stages:
 
-1.  **Conceptual**:
-    describe who the lesson is for,
-    what its overall goals are,
+1.  **Conteptual:** describe target audience,
+    overall lesson's goals,
     and how long it is going to be.
```

Comment:
I'm not sure I understand  what `how it is going to be` so I don't touch it here